.github/ISSUE_TEMPLATE/documentation_issue.md

```diff
@@ -0,0 +1,31 @@
+---
+name: 📗 Documentation Issue
+about: Documentation is unclear, or otherwise insufficient/misleading, examples aren't working as expected
+title: ""
+labels: ""
+assignees: ""
+---
+
+<!--
+If you don't follow the issue template, your issue may be closed.
+Please note this is an issue tracker, not a support forum.
+For general questions, please use StackOverflow.
+-->
+
+## Issue Creation Checklist
+
+[ ] I have read the [contribution guidelines](https://github.com/sequelize/sequelize/blob/main/CONTRIBUTING.md)
+
+## Issue Description
+
+### What was unclear/insufficient/not covered in the documentation
+
+Write here.
+
+### If possible: Provide some suggestion on how we can enhance the docs
+
+Write here.
+
+### Additional context
+
+Add any other context or screenshots about the issue here.
```

Comment:
The contributing docs provide almost no useful information for this type of issue. I've inline the only paragraph that was applicable instead of linking to it.

I've also removed the "write here" comments. If you decide to keep them, add them as comments instead

And I made the explanations for Additional context comments instead of text

```suggestion
## Issue Description

<--
Try to be as clear as possible. Don't assume that the maintainers will immediately understand the problem. Write your issue in a way that new contributors can also help (add links to helpful resources when applicable).
-->

### What was unclear/insufficient/not covered in the documentation



### If possible: Provide some suggestion on how we can enhance the docs



### Additional context

<-- Add any other context or screenshots about the issue here. -->
```